src/DynamoCoreWpf/UI/Themes/Modern/SidebarGridStyleDictionary.xaml

```diff
@@ -80,6 +84,49 @@
 
     <!--EndRegion-->
 
+    <!-- Region LibrarySearchView + IncanvasSearch-->
```

Comment:
Next code is common for LibrarySearchView and IncanvasSearch. I move it in style dictionary.